tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -110,6 +110,34 @@ def wrapper(self, *args, **kwargs):
     return _decorator
 
 
+def item_must_be_of_type(**params):
```

Comment:
I think this whole thing can be replaced with just an `isinstance` or `issubclass` check in the functions that need to check for type-ness -- that also lets us expand the types more easily in the future.

But I'm also not totally sure why it's necessary. If you just need to know the type to build the xml properly, we had a wrapper type for `Target` which you could borrow here (ScheduableItem or something) or just do type(THING).__name__

Reply:
I'm not sure I follow what you mean.  Could you drop by if you have a moment?